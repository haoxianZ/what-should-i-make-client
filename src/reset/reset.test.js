import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Reset from './reset';

describe(`About component`, () => {
  it('renders without crash', () => {
    const match={params:{userId:1}};

    const wrapper = shallow(<Reset match={match}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
}) 