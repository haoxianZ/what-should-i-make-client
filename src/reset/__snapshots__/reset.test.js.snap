// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders without crash 1`] = `
<section
  className="reset"
>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="resetCode"
    >
      Reset Code:
    </label>
    <input
      id="resetCode"
      name="resetCode"
      required={true}
      type="text"
    />
    <button
      className="restBtn"
      type="submit"
    >
      Submit
    </button>
  </form>
</section>
`;
